### PR TITLE
Compress screenshots in AWS package

### PR DESCRIPTION
**Pull request by kaiyan-sheng**
## What does this PR do?

This PR is to compress all screenshots in AWS package to reduce the overall size. 
The command I used is:
```
kaiyansheng@KaiyanMacBookPro:~/go/src/github.com/elastic/integrations/packages/aws/img (compress_dashboards)$ pngquant --ext .png --force *.png
```

`pngquant` can be installed with: `brew install pngquant`

Before:
```
-rw-r--r--   1 kaiyansheng  staff   393K Oct 13 10:19 filebeat-aws-cloudtrail.png
-rw-r--r--   1 kaiyansheng  staff   1.1M Oct 13 10:19 filebeat-aws-elb-overview.png
-rw-r--r--   1 kaiyansheng  staff   188K Oct 13 10:19 filebeat-aws-s3access-overview.png
-rw-r--r--   1 kaiyansheng  staff   2.5M Oct 13 10:19 filebeat-aws-vpcflow-overview.png
-rw-r--r--   1 kaiyansheng  staff   382K Oct 13 10:19 metricbeat-aws-billing-overview.png
-rw-r--r--   1 kaiyansheng  staff   257K Oct 13 10:19 metricbeat-aws-dynamodb-overview.png
-rw-r--r--   1 kaiyansheng  staff   1.1M Oct 13 10:19 metricbeat-aws-ebs-overview.png
-rw-r--r--   1 kaiyansheng  staff   598K Oct 13 10:19 metricbeat-aws-ec2-overview.png
-rw-r--r--   1 kaiyansheng  staff   597K Oct 13 10:19 metricbeat-aws-elb-overview.png
-rw-r--r--   1 kaiyansheng  staff   496K Oct 13 10:19 metricbeat-aws-lambda-overview.png
-rw-r--r--   1 kaiyansheng  staff   789K Oct 13 10:19 metricbeat-aws-overview.png
-rw-r--r--   1 kaiyansheng  staff   1.5M Oct 13 10:19 metricbeat-aws-rds-overview.png
-rw-r--r--   1 kaiyansheng  staff   329K Oct 13 10:19 metricbeat-aws-s3-overview.png
-rw-r--r--   1 kaiyansheng  staff   717K Oct 13 10:19 metricbeat-aws-sns-overview.png
-rw-r--r--   1 kaiyansheng  staff   347K Oct 13 10:19 metricbeat-aws-sqs-overview.png
-rw-r--r--   1 kaiyansheng  staff   557K Oct 13 10:19 metricbeat-aws-usage-overview.png
-rw-r--r--   1 kaiyansheng  staff   657K Nov 11 08:11 metricbeat-aws-s3-storage-lens-overview.png
-rw-r--r--   1 kaiyansheng  staff   604K Dec 13 14:02 filebeat-aws-firewall-alerts.png
-rw-r--r--   1 kaiyansheng  staff   673K Dec 13 14:02 filebeat-aws-firewall-flows.png
-rw-r--r--   1 kaiyansheng  staff   509K Dec 13 14:02 filebeat-aws-firewall-overview.png
-rw-r--r--   1 kaiyansheng  staff   422K Dec 13 14:02 metricbeat-aws-firewall-overview.png
```

After:
```
-rw-r--r--   1 kaiyansheng  staff   136K Mar  3 21:09 filebeat-aws-cloudtrail.png
-rw-r--r--   1 kaiyansheng  staff   401K Mar  3 21:09 filebeat-aws-elb-overview.png
-rw-r--r--   1 kaiyansheng  staff   169K Mar  3 21:09 filebeat-aws-firewall-alerts.png
-rw-r--r--   1 kaiyansheng  staff   186K Mar  3 21:09 filebeat-aws-firewall-flows.png
-rw-r--r--   1 kaiyansheng  staff   152K Mar  3 21:09 filebeat-aws-firewall-overview.png
-rw-r--r--   1 kaiyansheng  staff    55K Mar  3 21:09 filebeat-aws-s3access-overview.png
-rw-r--r--   1 kaiyansheng  staff   743K Mar  3 21:09 filebeat-aws-vpcflow-overview.png
-rw-r--r--   1 kaiyansheng  staff   101K Mar  3 21:09 metricbeat-aws-billing-overview.png
-rw-r--r--   1 kaiyansheng  staff    77K Mar  3 21:09 metricbeat-aws-dynamodb-overview.png
-rw-r--r--   1 kaiyansheng  staff   318K Mar  3 21:09 metricbeat-aws-ebs-overview.png
-rw-r--r--   1 kaiyansheng  staff   160K Mar  3 21:09 metricbeat-aws-ec2-overview.png
-rw-r--r--   1 kaiyansheng  staff   156K Mar  3 21:09 metricbeat-aws-elb-overview.png
-rw-r--r--   1 kaiyansheng  staff   125K Mar  3 21:09 metricbeat-aws-firewall-overview.png
-rw-r--r--   1 kaiyansheng  staff   144K Mar  3 21:09 metricbeat-aws-lambda-overview.png
-rw-r--r--   1 kaiyansheng  staff   214K Mar  3 21:09 metricbeat-aws-overview.png
-rw-r--r--   1 kaiyansheng  staff   356K Mar  3 21:09 metricbeat-aws-rds-overview.png
-rw-r--r--   1 kaiyansheng  staff   102K Mar  3 21:09 metricbeat-aws-s3-overview.png
-rw-r--r--   1 kaiyansheng  staff   164K Mar  3 21:09 metricbeat-aws-s3-storage-lens-overview.png
-rw-r--r--   1 kaiyansheng  staff   165K Mar  3 21:09 metricbeat-aws-sns-overview.png
-rw-r--r--   1 kaiyansheng  staff    88K Mar  3 21:09 metricbeat-aws-sqs-overview.png
-rw-r--r--   1 kaiyansheng  staff   148K Mar  3 21:09 metricbeat-aws-usage-overview.png
```

### CODE REVIEW

**Comment by andrewkroh** [comment]
What command did you use in case I wanted to replicate this?

File: packages/aws/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "1.12.2"
+  changes:
+    - description: Compress dashboard screenshots.
+      type: enhancement
```
Comment (mtojek) [inline packages/aws/changelog.yml:5]: if this is an enhancement, then maybe bump up the `minor` instead of `patch`?

**Review by mtojek — APPROVED** [review]
Left one nit-pick. Thanks for fixing this!

File: packages/aws/changelog.yml (hunk 1 above)
Comment (kaiyan-sheng) [reply packages/aws/changelog.yml:5]: 😬 Thanks for taking a look! Will do! (as you can see, I debated between bugfix or enhancement and ended up with half and half 😂 )

**Comment by kaiyan-sheng** [comment]
> What command did you use in case I wanted to replicate this?

@andrewkroh `pngquant --ext .png --force *.png` is the one I used. Let me add it in the PR description.

**Review by aspacca — APPROVED** [review]

**Pull request merged**